presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitManager.java

```diff
@@ -95,6 +95,7 @@
     private final int splitLoaderConcurrency;
     private final boolean recursiveDfsWalkerEnabled;
     private final CounterStat highMemorySplitSourceCounter;
+    private final String inputPathFilterClass;
```

Comment:
nit: How about making this `Optional`? Or even, can we create `PathFilter` here?